### DISCUSSION TITLE
Snyk

### DISCUSSION THREAD

**Discussion by ThomasvanderWal** [community/community#34675] [Code Security] [answered]
Hello,
What is snyk and why does snyk has access to my account?

**Comment by CaedenPH** [comment, marked as answer]
> Hello, What is snyk and why does snyk has access to my account?

Snyk is a developer security platform. You potentially logged in / created an account with your github account in order to protect your project, in which case you got an email from github letting you know that the snyk account you registered to is accessing your github account. 

Assuming this was you, you have nothing to worry about, otherwises if you believe you haven't tried to set up anything related to snyk, you should change your password and assume your github account is compromised.
